objects/45e367/contained/09004a/script.lua

```diff
@@ -14,6 +17,27 @@ function modifyThresholds(params)
     return modifyElements({color = params.color, elements = params.elements})
 end
 
+-- Edits a Fire element bag to be Dark Fire
+function makeDarkFire(bag)
+    bag.setName(darkFireTokenName)
+    bag.setColorTint(darkFireTokenColor)
+
+    local data = bag.getData().ContainedObjects[1]
+    data.Nickname = darkFireTokenName
+    data.ColorDiffuse.r = darkFireTokenColor.r
+    data.ColorDiffuse.g = darkFireTokenColor.g
+    data.ColorDiffuse.b = darkFireTokenColor.b
+    table.insert(data.Tags, "Moon")
```

Comment:
let's go ahead and remove this tag

Reply:
Not put the Moon tag on the Dark Fire element markers, you mean? But then they won't count towards the choice event element counter when tossed out in front of the play area. That's the entire point of the element markers having tags (besides Elemental Invocation, which people shouldn't be using Dark Fire markers for).